servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcRouter.java

```diff
@@ -161,8 +161,10 @@ private static void populateRoutes(final ExecutionContext executionContext,
             final String path = entry.getKey();
             final ServiceAdapterHolder adapterHolder = entry.getValue().buildRoute(executionContext);
             final StreamingHttpService route = closeable.append(adapterHolder.adaptor());
-            verifyNoOverrides(allRoutes.put(path, adapterHolder.serviceInvocationStrategy()
-                    .offloadService(executionContext.executor(), route)), path, emptyMap());
+            verifyNoOverrides(allRoutes.put(path,
+                    StreamingHttpService.offloadService(adapterHolder.serviceInvocationStrategy(),
+                            executionContext.executor(), Boolean.TRUE::booleanValue, route)),
```

Comment:
It could. I didn't "upgrade" them while making the rest of the renovations to avoid adding more potential problems to the mix.

Now done for all but the test cases which want to ensure that the offloading actually takes place so need to be unconditional